webpack/tableFetcher.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import Table from './Table'
+import LHLoader from './loader'
+
+const LH_API_ENDPOINT = 'https://api.lh.imesec.ime.usp.br/hacks'
+
+class TableFetcher extends Component {
+    constructor () {
+        super()
+        this.state = {
+            loaded: false,
+            success: false,
+            hacks: null,
+            nextDay: null,
+        }
+        this.fetchData()
+    }
+
+    fetchData () {
+        fetch(LH_API_ENDPOINT, { mode: 'cors' })
+        .then(res => res.json())
+        .then(res => {
+            console.log(res)
```

Comment:
This console.log is not necessary, but i agree on keeping it on the .catch() branch